src/main/java/uk/gov/hmcts/reform/pbis/servicebus/MessageQueueProcessor.java

```diff
@@ -73,21 +73,37 @@ private void processMessages(IServiceBusClient serviceBusClient) {
             messageCount++;
 
             MessageProcessingResult processingResult = processMessage(message);
+            updateMessageInSubscription(message, processingResult, serviceBusClient);
+            logProcessingResult(processingResult, message);
 
-            if (processingResult.resultType == SUCCESS) {
-                completeMessage(message, serviceBusClient);
-            } else {
+            if (processingResult.resultType == MessageProcessingResultType.SUCCESS) {
                 failureCount++;
```

Comment:
increase failure count on success?

Reply:
Well spotted. It was supposed to be `!+`